PalindromeLab/src/driver.py

```diff
@@ -0,0 +1,128 @@
+import random
+import palindrome
+import time
+from tests import testCases
+
+'''
+Tests that the user's answers are valid"
+1. The user returns a boolean value
+2. The user returns the correct boolean value
+'''
+def isValid(input, answer):
+    # check that the answer is bool
+    if isinstance(answer, bool):
+        # check that the answer is correct
+        strNum = str(input)
+        strNumOpp = strNum[::-1]
+        if strNumOpp == strNum:
+            if not answer:
+                return False
+            return True
+        else:
+            if not answer:
+                return True
+            return False
+    return False
+
+
+'''
+Runs a series of randomly generated numbers and checks if the
+user has a correct answer
+'''
+def randomTest(numTests):
+    score = 0
+    for i in range(numTests):
+        # generate random integers
+        number = random.randint(0, 2**30-1)
+        res = palindrome.palindrome(number)
+        if isValid(number, res):
+            print(f"Passed test number {i} of {numTests}. Input: {number}, Result: {res}")
```

Comment:
Can you use i + 1 to avoid an off-by-one issue?

Currently it goes from 0/100 to 99/100